packages/aws-cdk-lib/aws-rds/lib/cluster.ts

```diff
@@ -965,14 +972,14 @@ class ImportedDatabaseCluster extends DatabaseClusterBase implements IDatabaseCl
   public readonly clusterIdentifier: string;
   public readonly connections: ec2.Connections;
   public readonly engine?: IClusterEngine;
-  public readonly secret?: secretsmanager.ISecret;
 
   private readonly _clusterResourceIdentifier?: string;
   private readonly _clusterEndpoint?: Endpoint;
   private readonly _clusterReadEndpoint?: Endpoint;
   private readonly _instanceIdentifiers?: string[];
   private readonly _instanceEndpoints?: Endpoint[];
 
+  public readonly secret?: secretsmanager.ISecret;
```

Comment:
We should leave the variable declaration grouped with the other `public` variables as before

Reply:
@lpizzinidev i was not sure which way was better, but make sense splitting it for readability. Done.